lib/models/subscription.js

```diff
@@ -24,6 +24,10 @@ module.exports = (sequelize, DataTypes) => {
     reviews: false,
   };
 
+  const FixedSettings = {
```

Comment:
This should probably have a better name - and maybe not even include the word 'settings'?

Reply:
Hmm, should this be the same setting as `public` events? It looks like they're included in the same `repository` webhook: https://developer.github.com/v3/activity/events/types/#repositoryevent.

Reply:
the `public` event actually seems to come in via two webhooks. One as part of the repository event and one as part of the public event: https://developer.github.com/v3/activity/events/types/#publicevent

Right now we listen to the public event